tools/pythonpkg/tests/arrow/test_timestamps.py

```diff
@@ -0,0 +1,51 @@
+import duckdb
+import os
+import sys
+import datetime
+import pytest
+try:
+    import pyarrow as pa
+    import pandas as pd
+    can_run = True
+except:
+    can_run = False
+
+class TestArrowTimestamps(object):
+    def test_timestamp_types(self, duckdb_cursor):
+        if not can_run:
+            return
+        data = (pa.array([datetime.datetime.now()], type=pa.timestamp('ns')),pa.array([datetime.datetime.now()], type=pa.timestamp('us')),pa.array([datetime.datetime.now()], pa.timestamp('ms')),pa.array([datetime.datetime.now()], pa.timestamp('s')))
+        arrow_table = pa.Table.from_arrays([data[0],data[1],data[2],data[3]],['a','b','c','d'])
```

Comment:
Can we have some tests with these timestamp types to pandas data frames as well?